### DISCUSSION TITLE
Error scanning qr code

### DISCUSSION THREAD

**Discussion by JoshuaBarawa** [devlikeapro/waha#132] [Q&A]
![WhatsApp Image 2023-07-07 at 08 24 55](https://github.com/devlikeapro/whatsapp-http-api/assets/68270558/87c61440-1942-4bab-a58b-3e1be7ba0276)
![WhatsApp Image 2023-07-07 at 08 19 51](https://github.com/devlikeapro/whatsapp-http-api/assets/68270558/07b87472-5c4a-4c6e-a413-81c148b70e34)
Hi, Why do I get these errors sometimes when I scan the QR Code? And is there a way I can fix this issue?

thanks

**Comment by allburov** [comment]
Hm, never seen it before. Is there any logs for that in the docker logs?

**Reply by JoshuaBarawa** [reply]
Nothing from the logs just a bunch of generated QR codes. I have tried generating the QR code again but this is what I'm getting.
![Screenshot from 2023-07-07 15-17-41](https://github.com/devlikeapro/whatsapp-http-api/assets/68270558/182e3dbb-6f2a-4fe7-bd91-de9c9d0a9573)
Also, it seems I cannot stop or log out of thee session.

**Comment by operatortt** [comment]
Same problem nowadays. Could you solve it? If yes how please.. :)

**Reply by allburov** [reply]
Hi! Does it happen with all QR?

Could you try to update WhatsApp on the phone to the latest version and try using the latest WAHA Plus? 
https://www.patreon.com/posts/waha-plus-update-100765474

We'll release the fix in WAHA Core on April 1!

**Reply by operatortt** [reply]
Changed whatsapp business from Android to IOS
now it works :)
Thanks
